Lib/Data.js

```diff
@@ -0,0 +1,56 @@
+const pets = [
+  {
+    id: 1,
+    name: "Johnny",
+    type: "Turtle",
+    picture: "🐢",
+    characteristics: ["smart ", "joyful"],
```

Comment:
Please do not include a space in your data here. You can add the space, when you render the data.